From 3f1ee1917adf4188a0c4fb9b78626cff1c49cea8 Mon Sep 17 00:00:00 2001
From: nopeless <38830903+nopeless@users.noreply.github.com>
Date: Sun, 18 Jun 2023 17:08:43 +0900
Subject: [PATCH] fix: ts issue

---
 src/url.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/url.ts b/src/url.ts
index 89e8a1d8..ac30c8cd 100644
--- a/src/url.ts
+++ b/src/url.ts
@@ -1,6 +1,6 @@
 import { AbsoluteURL, ReadonlyURL } from './url/format';
 
-export { StandardURL, standardize } from './url/format';
+export { type StandardURL, standardize } from './url/format';
 export { ReadonlyURL } from './url/format';
 
 type Widen<T> = T extends `${infer _}` ? string : T;
-- 
2.39.1.windows.1

